### PR TITLE
EN-1858 Modify content attribute to store and show image metadata in content (version 4.3.4)

### PR DESCRIPTION
**Pull request by eugeniosant**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

File: admin-console/src/main/java/com/agiletec/plugins/jacms/apsadmin/content/ContentAdminAction.java [hunk 1]
```diff
@@ -1,126 +1,238 @@
...
+     * referenze tra Contenuti e Pagine, Risorse, Gruppi e Contenuti stessi in
+     * apposita tabella (elementi a servizio di controlli di autorizzazione e
+     * validazione).
+     *
+     * @return Il codice del risultato dell'azione.
+     */
+    public String reloadContentsReference() {
+        try {
+            String result = this.configSystemParams();
+            if (!result.equals(SUCCESS)) {
```
Comment (joewhite101) [inline admin-console/src/main/java/com/agiletec/plugins/jacms/apsadmin/content/ContentAdminAction.java]: Just to check is this return correct? If the first one fails you don't want the contentManager call?

File: engine/src/main/db/update/scriptPort.sql [hunk 2]
```diff
@@ -1,5 +1,14 @@
+-- EN-1748
 ALTER TABLE pages_metadata_draft ADD COLUMN groupcode character varying(30);
 ALTER TABLE pages_metadata_online ADD COLUMN groupcode character varying(30);
 UPDATE pages_metadata_online SET groupcode = (SELECT pages.groupcode FROM pages WHERE pages_metadata_online.code = pages.code);
 UPDATE pages_metadata_draft SET groupcode = (SELECT pages.groupcode FROM pages WHERE pages_metadata_draft.code = pages.code);
 ALTER TABLE pages DROP groupcode;
+
+-- EN-1858
+INSERT INTO sysconfig (version, item, descr, config) VALUES ('production', 'jacms_resourceMetadataMapping', 'Mapping between resource Metadata and resource attribute fields', '<mapping>
```
Comment (joewhite101) [inline engine/src/main/db/update/scriptPort.sql]: Will this load automatically for CDP or do we have something else to do when we give them 4.3.4?

File: admin-console/src/main/java/com/agiletec/plugins/jacms/apsadmin/content/ContentAdminAction.java (hunk 1 above)
Comment (eugeniosant) [reply admin-console/src/main/java/com/agiletec/plugins/jacms/apsadmin/content/ContentAdminAction.java]: yes, it's only to check the first call; you could eliminate this check (very difficult for it to fail), I put it to be sure

**Comment by joewhite101** [comment]
https://confluence.entando.org/display/PP/Porting+from+5.0.0+to+5.1.0

**Pull request merged**